build/modules/sdk/build.xml

```diff
@@ -90,7 +90,7 @@
                         src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/log4j-1.2.14.jar" />
             <zipfileset includes="**"
                         excludes="commons-logging.properties META-INF/MSFTSIG.RSA META-INF/MSFTSIG.SF"
-                        src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/applicationinsights-core-0.9.5.jar" />
+                        src="${dir.binaries}/${name.dist-directory.sdk}/redist/lib/applicationinsights-core-1.0.4-SNAPSHOT.jar" />
```

Comment:
Shouldn't we rename the JAR removing SNAPSHOT? I doubt that it's OK to ship snapshots. We also might have to take some special care of signing this JAR. Was the original 0.9.5 signed before we included it into the repository?